canonical-web-and-design/vanilla-framework
# Handle text overflow on expanding table

## Done

- Updated expanding table to behave more like other table examples: 
  - removed `word-break: break-all` rule
  - added `text-overflow: ellipsis` and `overflow: hidden` to all `th` and `td` elements, so there's no need to use `u-truncate`
  - switched `display` to `block` (`text-overflow` seems to only apply to [block elements](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow#formal_definition))

Fixes #2697 

## QA

- Open [demo](/docs/examples/patterns/tables/table-expanding)
- Reduce the width of the browser, and see that the text in the column both wraps and displays an ellipsis when the word is too long to fit

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
